src/Sylius/Behat/Resources/config/services/elements/admin/product.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <service
+            id="sylius.behat.element.admin.product_association_type.form"
+            class="Sylius\Behat\Element\Admin\ProductAssociationType\FormElement"
+            parent="sylius.behat.element.admin.crud.form"
+        >
+            <argument type="service" id="Sylius\Behat\Service\Helper\AutocompleteHelperInterface" />
+        </service>
+
+        <service
+            id="sylius.behat.element.admin.product_association.form"
+            class="Sylius\Behat\Element\Admin\Product\ProductAssociationsFormElement"
```

Comment:
To be consistent with the existing ones:
```suggestion
            id="sylius.behat.element.admin.product.association_form"
            class="Sylius\Behat\Element\Admin\Product\AssociationsFormElement"
```

Reply:
https://github.com/Sylius/Sylius/pull/16349/files#diff-44fb12c6ada676847994567d9163b245dfacccbe88d9de9e3a2fa2ee512030b8R28